### ISSUE TITLE
ROS2: use xacro from apt

### ISSUE THREAD

**Issue by AAlon** [aws-robotics/aws-robomaker-sample-application-cloudwatch#55]
See https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/pull/54

**Comment by timrobotson** [comment]
xacro isn't needed any more after #56 as the urdf files are pre-built from xacro and committed to the repository now.

**Comment by mm318** [comment]
Closing this issue as `xacro` is no longer needed after https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/pull/56.
